pkg/api/versions.go

```diff
@@ -155,14 +156,28 @@ func yamlToStub(data []byte) ([]*Stub, error) {
 			}
 			return stubs, err
 		}
-		stubs = append(stubs, stub)
+		expandList(&stubs, stub)
 	}
 	if stubs == nil && len(errs) > 0 {
 		return nil, fmt.Errorf("one or more errors parsing yaml resulted in no versions found: %v", errs)
 	}
 	return stubs, nil
 }
 
+// expandList checks if we have a List manifest.
+// If it is the case, the manifests inside are expanded, otherwise we just return the single manifest
+func expandList(stubs *[]*Stub, currentStub *Stub) {
+	if currentStub.Items != nil {
+		klog.V(8).Infof("We found a list with %d items, let's expand it", len(currentStub.Items))
```

Comment:
nit: Typically we try not to capitalize log messages and make them purely informative.

```suggestion
		klog.V(5).Infof("found a list with %d items, attempting to expand", len(currentStub.Items))
```

Reply:
thanks, fixed